File: BetSystem.API/Models/Bet.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace BetSystem.API.Models
{
    [Table("Bets")]
    public class Bet
    {
        public int Id { get; set; }
        public virtual Team Team { get; set; }
        public int TeamId { get; set; }

        [StringLength(50)]
        public string Home { get; set; }

        [StringLength(50)]
        public string Away { get; set; }
        public DateTime Date { get; set; }
        public double Odd { get; set ; }
        public double Stake { get; set; }
        public double CashReturn { get; set; }
        public bool? Won { get; set; }
        public double Withdrawal { get; set; }
        public double Profit { get; set; }
    }
}

File: BetSystemSPA/src/app/_models/bet.ts
export interface bet{
    id: number;
    teamId: number;
    team: string;
    country: string;
    home: string;
    away: string;
    date: Date;
    // date_text: string;
    odd: number;
    stake: number;
    cashReturn: number;
    won: boolean;
    withdrawal: number;
    profit:number;
}